src/mumble/AudioOutputSpeech.cpp

```diff
@@ -288,14 +291,17 @@ bool AudioOutputSpeech::needSamples(unsigned int snum) {
 						memset(pOut, 0, sizeof(float) * iFrameSize);
 				} else if (umtType == MessageHandler::UDPVoiceOpus) {
 #ifdef USE_OPUS
-					decodedSamples = opus_decode_float(opusState,
+					if (opusState) {
```

Comment:
This was not here before? What is this?

Reply:
Additional check: if `opusState` is not initialized, the function makes the program crash.

Reply:
When is this NULL but oCodec is not?

The funciton `opus_decoder_create` does not seem to define when it returns `NULL`, and what it does in error states. Can we do that?
Maybe we should be checking the error code that the function returns and set NULL ourselves?

Reply:
> Maybe we should be checking the error code that the function returns and set NULL ourselves?

Yeah, it would be better.

Do you prefer to do it in this pull request or in a dedicated one?

Reply:
Let's do it in another pull request.